src/integrationTest/sam.test.ts

```diff
@@ -374,9 +374,9 @@ describe('SAM Integration Tests', async function () {
                     // Allow previous sessions to go away.
                     await waitUntil(
                         async function () {
-                            return vscode.debug.activeDebugSession === undefined
+                            return (vscode.debug.activeDebugSession === undefined) ? true : undefined
```

Comment:
The interface to `waitUntil()` is now rather awkward :/ This is why I originally checked "falsey" instead of strict `undefined`. Was the gain worth this ergnomic cost?